zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1706,26 +1721,45 @@ components:
           format: int64
         tenantId:
           type: string
-    AuthorizationAssignRequest:
+    AuthorizationPatchRequest:
       type: object
       properties:
-        ownerKey:
-          type: integer
-          format: int64
-        ownerType:
+        action:
+          description: Indicates if permissions should be added or removed.
           type: string
           enum:
+            - ADD
+            - REMOVE
+        resourceType:
+          description: The type of resource to add/remove perissions to/from.
+          enum:
+            - AUTHORIZATION
+            - MESSAGE
+            - JOB
+            - APPLICATION
+            - TENANT
+            - DEPLOYMENT
+            - PROCESS_DEFINITION
+            - USER_TASK
+            - DECISION_REQUIREMENTS_DEFINITION
+            - DECISION_DEFINITION
+            - USER_GROUP
             - USER
             - ROLE
-            - GROUP
-        resourceKey:
-          type: string
-        resourceType:
-          type: string
         permissions:
           type: array
           items:
-            type: string
+            properties:
+              permissionType:
+                enum:
+                  - CREATE
+                  - READ
+                  - UPDATE
+                  - DELETE
```

Comment:
❓ As per the [migration doc](https://docs.google.com/document/d/1yShs_FRhrFv8NmyO94bElZk9UPuiMYGfbKUhn5vxkDE/edit#heading=h.jyh8xap11g2r) should we have a wildcard here?

Reply:
This is the first time I'm seeing this doc. You tell me if we need a wildcard 😄 I've been using [this doc](https://docs.google.com/document/d/1ikxw9ARCv-MBwASnj7WhW5uHCndCRk2B35xi-cB2N0M/edit#heading=h.25pdsyjx8m33) as my guideline.

Reply:
Ah, I think for now we will leave the wildcard out, it adds complexity and can just be achieved right now by selecting all the permissions. Down the line we may want to add one, but lets leave it